defaults-themes.php

```diff
@@ -0,0 +1,100 @@
+<?php
+function custom_admin_styles() {
+  ?>
+  <style>
+    /* .cf-select__input option[value=nad-denim] {
+      display: none !important;
+    } */
+
+    /* .cf-select__input option[value=nad-nile] {
+      display: none !important;
+    } */
+
+    /* .cf-select__input option[value=nad-amethyst] {
+      display: none !important;
+    } */
+
+    /* .cf-select__input option[value=nad-spark] {
+      display: none !important;
+    } */
+
+    /* .cf-select__input option[value=nad-miracle] {
+      display: none !important;
+    } */
+
+    /* .cf-select__input option[value=nad-branch] {
+      display: none !important;
+    } */
+
+    /* .cf-select__input option[value=nad-vine] {
+      display: none !important;
+    } */
```

Comment:
Please remove the comments for all `nad-` color options so it only shows the traditional color options.